livekit_egress.proto

```diff
@@ -230,8 +230,17 @@ message StreamInfoList {
 }
 
 message StreamInfo {
+  enum Status {
+    ACTIVE = 0;
+    FINISHED = 1;
+    FAILED = 2;
+  }
+
   string url = 1;
+  int64 started_at = 2;
```

Comment:
we had a gap here? what were the fields used for before?

Reply:
started_at and ended_at, which I removed a while back for duration. Would probably be helpful though since the streams can have different start and end times